app-config.yaml

```diff
@@ -89,6 +89,18 @@ catalog:
         #   token:
         #     $secret:
         #       env: GHE_PRIVATE_TOKEN
+    githubOrg:
```

Comment:
Curious. Is this going to be moved to the new readers concept? I think we'd like to tap into the octocat/graphql client as well so might be nice to have this config less specific to the processor and closer to the underlying util lib. Thoughts?

Reply:
My thinking is that we can go a bit verbose for now and duplicate config, but then do a proper cleanup once we bring in GitHub Apps, as that is likely gonna have a pretty big impact on how things are tied together and might need some additional plumbing